includes/Admin/Google_Fonts.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Class Google_Fonts.
+ *
+ * Registers Google fonts for admin screens.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Admin;
+
+use Google\Web_Stories\Infrastructure\Conditional;
+use Google\Web_Stories\Infrastructure\Registerable;
+use Google\Web_Stories\Infrastructure\Service;
+use WP_Styles;
+
+/**
+ * Class Google_Fonts
+ *
+ * Enqueue Google Fonts stylesheet.
+ *
+ * @package Google\Web_Stories\Admin
+ */
+class Google_Fonts implements Conditional, Service, Registerable {
+
+	const HANDLE = 'web-stories-fonts';
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+		return is_admin() && ! wp_doing_ajax();
+	}
+
+	/**
+	 * Provides the asset handle.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return string
+	 */
+	public function get_handle() {
```

Comment:
I don't think this method is needed.

Any consumer could just use `($this->google_fonts )::HANDLE`

Reply:
I got a lint error in PHPStorm when I did something similar.

Reply:
`$this->google_fonts::HANDLE` will cause warnings due to using PHP7+ syntax. But `($this->google_fonts )::HANDLE` should work. Can we try that?

Reply:
I tried it. But it still failed in 5.6. See https://github.com/google/web-stories-wp/runs/2676667181

If you reference [Class Constants ](https://www.php.net/manual/en/language.oop5.constants.php). Even the examples have getter methods..

Reply:
Apparently one needs to do stuff like this on PHP 5:

```php
$fonts = $this->google_fonts;
echo $fonts::HANDLE
```

Let's keep the method for now but revisit in #7597

Reply:
Agreed, this is a small detail, it should not hold up the PR.